src/services/picture-uploader.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { PicturesStore } from 'api/collections';
+import { Picture } from 'api/models';
+import { UploadFS } from 'meteor/jalik:ufs';
+import { _ } from 'meteor/underscore';
+
+@Injectable()
+export class PictureUploader {
+  select(): Promise<File> {
+    return new Promise((resolve, reject) => {
+      UploadFS.selectFile((file: File) => {
+        resolve(file);
+      });
+    });
+  }
+
+  upload(file: File): Promise<Picture> {
```

Comment:
I have no use for other callbacks, therefore I didn't want to make things unnecessarily complicated.